docs/differences.md

```diff
@@ -7,13 +7,46 @@ sidebar_position: 3
+
+| Item             | Ethereum                                                                                                                                                                | Kakarot       |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| ecAdd            | Point addition (ADD) on the elliptic curve `alt_bn128`                                                                                                                  | Not supported |
+| ecMul            | Scalar multiplication (MUL) on the elliptic curve `alt_bn128`                                                                                                           | Not supported |
+| ecPairing        | Bilinear function on groups on the elliptic curve `alt_bn128`                                                                                                           | Not supported |
+| Point evaluation | Verify p(z) = y given commitment that corresponds to the polynomial p(x) and a KZG proof. Also verify that the provided commitment matches the provided versioned_hash. | Not supported |
+
+## EVM State computation
+
+The below state computation differs in behavior between Ethereum and Kakarot:
+
+| Item                   | Ethereum                                                     | Kakarot                                                                                                                                                                                                          |
+| ---------------------- | ------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Block hash computation | keccak256(RLP.encode(Block Header))                          | [pedersen(Starknet Block Header)](https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/header/)                                                                                 |
+| State root computation | Root of state keccak-MPT                                     | [poseidon(root of contract pedersen-MPT, root of contract class pedersen-MPT)](https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/starknet-state/)                            |
 | Transaction receipt    | Transaction receipt is available once a transaction is mined | Transaction receipt is available once a transaction is processed and guaranteed to be mined. This means that blockhash field of the receipt is optional and will be non-null only once the transaction is mined. |
+
+## Transaction reversion
+
+In Ethereum, a transaction can be reverted if validation parameters are
```

Comment:
this whole paragraph is not so easy to understand 

i'm not sure how to modify it

Reply:
ok let me know